src/routers/HelloWorldRouter.js

```diff
@@ -12,14 +11,12 @@ class HelloWorldRouter extends ApplicationRouter<string> {
     return '/';
   }
 
-  async content(req: Request): Promise<string> {
-    const checkInit = (req.query.awaitInit !== undefined && `${await init}.`) || process.env.PORT;
-
-    return (`Hello World! Environment: ${process.env.NODE_ENV || 'unknown'} | `
+  // eslint-disable-next-line require-await
```

Comment:
don't know what this means but okay

Reply:
it's basically because it's a subclass and in the parent class it's defined as an `async` function, but in this specific instance we don't need the `async` functionality, if that makes any sense
so we ignore the linter error